File: 5kyi/human_readable_time.py
#!/usr/bin/env python3
# https://www.codewars.com/kata/52685f7382004e774f0001f7


def make_readable(total_seconds: int) -> str:
    hours = total_seconds // 3600
    minutes = total_seconds // 60 % 60
    seconds = total_seconds % 60
    return "{:02d}:{:02d}:{:02d}".format(hours, minutes, seconds)

File: codewars/5kyi/human_readable_time.lua
#!/usr/bin/env lua
-- https://www.codewars.com/kata/52685f7382004e774f0001f7
local solution = {}
function solution.make_readable(seconds)
  local result
  local hours = math.floor(seconds / 3600)
  local minutes = math.floor(seconds / 60) % 60
  seconds = seconds % 60
  return string.format("%02d:%02d:%02d", hours, minutes, seconds)
end
return solution